figures/README.md

```diff
@@ -78,6 +78,7 @@ RUN_LOCAL=1 bash ../scripts/run-analyses-for-figures.sh
 _⚠️ This requires 64GB of RAM to run successfully! You can set `RUN_LOCAL=1` for local testing that skips the RAM-intensive steps._
 
 The script [`generate-figures.sh`](./generate-figures.sh) runs all scripts in [`figures/scripts/`](./scripts/) to generate manuscript-ready figure panels, as well as copies certain panels from analyses that are meant for the manuscript.
+**Note that it is ASSUMED that all results, including certain files in `scratch/`, will have been generated with `../scripts/run-manuscript-analyses.sh` before running this script.**
```

Comment:
Should we add a check to the `generate-figures.sh` script? Or is the concern here that there will be old versions of the files hanging around? If so, I would make sure to say this directly.

If we want to get fancy, we could check in the figure script that the relevant files in `scratch/`  are not too old and print a warning if they are older than, say, a week.

Reply:
>  If so, I would make sure to say this directly.
👍 

> If we want to get fancy, we could check in the figure script that the relevant files in scratch/ are not too old and print a warning if they are older than, say, a week.

I think this is a good way to check!

Reply:
> though I will admit I did go through
Do you mean "did not"? Seems like doing it is not an admission :) 

I am pretty certain that the order is good, based on what results feed into other modules.